### PR TITLE
Update customer-key-overview.md

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Added note of clarification on usage of public folder, group and shared mailbox

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/customer-key-overview.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ A data encryption policy defines the encryption hierarchy to encrypt data using
 
 Later, you can either refresh the DEP or assign a different DEP to the mailbox as described in [Manage Customer Key for Office 365](customer-key-manage.md). Each mailbox must have appropriate licenses in order to assign a DEP. For more information about licensing, see [Before you set up Customer Key](customer-key-set-up.md#before-you-set-up-customer-key).
 
+Note:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-overview.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/compliance/customer-key-overview.md [hunk 2]
```diff
@@ -57,6 +57,9 @@ A data encryption policy defines the encryption hierarchy to encrypt data using
...
+The DEP can be applied to shared mailbox, public folder mailbox and Microsoft 365 group mailbox for tenants that meet the licensing requirement for user mailboxes, even though some of these mailbox types cannot be a assigned license (Public Folder mailbox and Microsoft 365 Group mailbox) or need license for increasing storage (Shared mailbox).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-overview.md]:
```suggestion
> The DEP can be applied to a shared mailbox, public folder mailbox, and Microsoft 365 group mailbox for tenants that meet the licensing requirement for user mailboxes, even though some of these mailbox types cannot be an assigned license (public folder mailbox and Microsoft 365 group mailbox) or need a license for increasing storage (shared mailbox).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@Batre-MSFT - please also change the note tag as indicated.

**Comment by Batre-MSFT** [comment]
Can some one confirm if the change was made to article? I don't see it yet

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Comment by KCCross** [comment]
I'd like Brijesh to review. Also, I'd like the content removed from "Note" as it's not secondary information, it's primary information.

**Review by brdesai — COMMENTED** [review]
Looks good. One suggestion: How about linking the terms "shared mailbox", "public folder mailbox" and "Microsoft 365 group mailbox" to their official documentation.

shared mailbox:
https://docs.microsoft.com/en-us/microsoft-365/admin/email/about-shared-mailboxes?view=o365-worldwide

public folder mailbox:
https://docs.microsoft.com/en-us/exchange/collaboration-exo/public-folders/public-folders

Microsoft 365 group mailbox:
https://docs.microsoft.com/en-us/microsoft-365/admin/create-groups/office-365-groups?view=o365-worldwide

File: microsoft-365/compliance/customer-key-overview.md [hunk 3]
```diff
@@ -57,6 +57,9 @@ A data encryption policy defines the encryption hierarchy to encrypt data using
 
 Later, you can either refresh the DEP or assign a different DEP to the mailbox as described in [Manage Customer Key for Office 365](customer-key-manage.md). Each mailbox must have appropriate licenses in order to assign a DEP. For more information about licensing, see [Before you set up Customer Key](customer-key-set-up.md#before-you-set-up-customer-key).
 
+> [!NOTE]
+> The DEP can be applied to a shared mailbox, public folder mailbox, and Microsoft 365 group mailbox for tenants that meet the licensing requirement for user mailboxes, even though some of these mailbox types cannot be an assigned license (public folder mailbox and Microsoft 365 group mailbox) or need a license for increasing storage (shared mailbox).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-overview.md:61]:
```suggestion
> The DEP can be applied to a [shared mailbox](https://docs.microsoft.com/microsoft-365/admin/email/about-shared-mailboxes?view=o365-worldwide), [public folder mailbox](https://docs.microsoft.com/exchange/collaboration-exo/public-folders/public-folders), and [Microsoft 365 group mailbox](https://docs.microsoft.com/microsoft-365/admin/create-groups/office-365-groups?view=o365-worldwide) for tenants that meet the licensing requirement for user mailboxes, even though some of these mailbox types cannot be an assigned license (public folder mailbox and Microsoft 365 group mailbox) or need a license for increasing storage (shared mailbox).
```

Comment (JohanFreelancer9) [reply microsoft-365/compliance/customer-key-overview.md:61]: As per @brdesai's suggestion (removed locals).

**Pull request merged**